scripts/sidebar/api/index.js

```diff
@@ -1,8 +1,15 @@
 const clientId =
   "37fb1728bc4d17ee3bd799a37941e9dd031e8889e0ae316a4ffc461c13b78265";
 
+// TODO: production key ?
+const apiKey = "eZzJ2ixv8CTOOE9yvR0L4x9P0IUuG8gq";
```

Comment:
We should probably rename those keys something like: `SPLASH_API_KEY` and `GIFY_API_KEY`